cloud/amazon/ec2_placement.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+DOCUMENTATION = '''
+---
+module: ec2_placement
+short_description: create, remove or list placement group(s) in EC2.
+description:
+    - Creates, removes and lists placement groups in EC2. This module has a dependency on python-boto.
+version_added: "2.0"
+options:
+  name:
+    description:
+      - The EC2 placement group name.
+    required: False
+    default: null
+    aliases: []
+  strategy:
+    description:
+      - The placement strategy of the new placement group. Currently, the only acceptable value is cluster
+      required: false
+    default: cluster
+    choices: ['cluster']
+    aliases: []
+  state:
+    description:
+      - Whether the placement group should be present or absent. Use list to show all or search specific placement group.
+    required: false
+    default: present
+    choices: ['present', 'absent', 'list']
```

Comment:
list is a bad pattern that some of the amazon modules already have, I would prefer to move this to the ec2_facts modules